### ISSUE TITLE
Do not hide cursor in client os

### ISSUE THREAD

**Issue by fantastxmw** [synergy/synergy#4470]
I used the nightly build version 1.7.1, windows 7 x64 server and osx 10.10.2 client.

After moved cursor from osx to windows, the cursor of osx was hidden. 

It's annoyance and broken my work.

Is it a bug or feature? How can I avoid the hidden?

**Comment by nbolton** [comment]
When you move the cursor from the client to the server, the cursor is hidden on the client? This is normal behavior. Do you think the cursor should remain visible at the edge of the screen?

**Comment by fantastxmw** [comment]
Yes. The mouse attached to the client os should work as normal. 




I set up a windows as synergy server and a OS X as client. When I was mainly working on windows, I use the mouse to control both os, it works great. But when I started to work on OS X and need to use touchpad, the hidden cursor causes problems. 




So if it is a normal feature, an option would be great.



—
Best regards，
William Xue

**Comment by speaker** [comment]
This is then a request for an option to keep the client pointer visible when the mouse leaves the client.

**Comment by nbolton** [comment]
@speaker I changed the title to line up with your comment.
